aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>lower case "next steps"</p><p> </p><p>features or preprocess --&gt; features and preprocessed</p><p> </p><p>it's --&gt; its</p><p> </p><p>"it is ambient as it is not limited by angles" ??? reference? what does this mean? I don't think you introduced this concept earlier.</p><p> </p><p>Sagemaker market place --&gt; SageMaker Marketplace</p><p> </p><p>task which --&gt; tasks that</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>